src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DefaultProjectSnapshot.cs

```diff
@@ -204,14 +202,16 @@ public DefaultProjectSnapshot WithComputedUpdate(ProjectSnapshotUpdateContext up
             return new DefaultProjectSnapshot(update, this);
         }
 
-        public bool HasChangesComparedTo(ProjectSnapshot original)
+        public bool HasChangesComparedTo(DefaultProjectSnapshot original)
         {
             if (original == null)
             {
                 throw new ArgumentNullException(nameof(original));
             }
 
-            return !object.Equals(Configuration, original.Configuration);
+            // We don't have any computed state right now, so treat all background updates as
+            // significant.
+            return !object.Equals(ComputedVersion, original.ComputedVersion);
```

Comment:
I decided to leave in the concept of computed state and the background worker code to compute it. I think we'll find more things to use it for (tag helpers). For now it seems fine to leave it and more destabilizing to delete it.